### DISCUSSION TITLE
How does the simulation statefile work?

### DISCUSSION THREAD

**Discussion by govertv** [asreview/asreview#597] [Q&A] [answered]
What happens when I start a new simulation run but use a state_file filename that already exists? It seems like it tries to continue with the simulation that was already in the state_file (and disregards all other CLI options). Can I use this to interrupt one of my simulation runs and continue it later?

**Comment by J535D165** [comment, marked as answer]
Yes, this is indeed the case. Yes, you can use this to interrupt. 
I can't guarantee that this always works. Might depend on the type of interruption. For example, a power shutdown might be problematic, but KeyboardInterrupt not. Love to hear about your experiences.
